CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ Order should be `CHANGE`, `FEATURE`, `ENHANCEMENT`, and `BUGFIX`
 
 ## unreleased/master
 
-[FEATURE] Support Arm64 on Darwin for all binaries (benchtool etc). #215
+* [FEATURE] Support Arm64 on Darwin for all binaries (benchtool etc). #215
+* [BUGFIX] Fix legends displaying wrong symbols for updates and deletions. #226
```

Comment:
Can you please add the commands this affects?

Reply:
@gotjosh which would that be? It does touch the `cortextool rules` command but also the generic file `pkg/printer/printer.go`. Should I glance over the latter and make it:
```
[BUGFIX] Fix `cortextool rules` legends displaying wrong symbols for updates and deletions. #226
```
Or rather prefix with the entry with `Cortextool:`?

Reply:
I made an educated guess based on previous changelog entries.

Reply:
This is perfect. Thank you very much for your contribution!